Assets/HoloToolkit/UI/Shaders/3DTextShader.shader

```diff
@@ -70,20 +72,22 @@
 
                 v2f vert (appdata_t v)
                 {
+                    UNITY_SETUP_INSTANCE_ID(v);
                     v2f o;
                     o.vertex = mul(UNITY_MATRIX_MVP, v.vertex);
                     o.texcoord = TRANSFORM_TEX(v.texcoord, _MainTex);
                     o.color = v.color;
                     #ifdef UNITY_HALF_TEXEL_OFFSET
                     o.vertex.xy += (_ScreenParams.zw-1.0)*float2(-1,1);
                     #endif
+                    UNITY_INITIALIZE_VERTEX_OUTPUT_STEREO(o);
                     return o;
                 }
 
                 half4 frag (v2f i) : COLOR
                 {
                     half4 col = i.color;
-                    col.a *= tex2D(_MainTex, i.texcoord).a;
+                    col.a *= tex2D(_MainTex, UnityStereoScreenSpaceUVAdjust(i.texcoord, _MainTex_ST)).a;
```

Comment:
The rest makes sense and looks good to me, but I haven't come across this function before, and its unclear to me if its necessary or not. Presumably it was if you added it, but I'm not sure what the exact purpose is.